packages/fuel-indexer/src/service.rs

```diff
@@ -60,6 +55,32 @@ impl IndexerService {
         mut manifest: Manifest,
     ) -> IndexerResult<()> {
         let mut conn = self.pool.acquire().await?;
+
+        if (queries::get_indexer_id(&mut conn, &manifest.namespace, &manifest.identifier)
```

Comment:
Same comment here with regard to readability 

```rust
```rust
let indexer_exists = queries::get_indexer_id(&mut conn, &namespace, &identifier).await;
if (indexer_exists) {
  // do the thing..
}
```